.github/ISSUE_TEMPLATE/4-constraint-work.yaml

```
@@ -0,0 +1,112 @@
+name: Create or Revise a Constraint
```

Minor, non-blocking: we may want to even take some away, yes?

```suggestion
name: Add, Change, or Remove a Constraint
```